src/native/hash/bitfn.h

```diff
@@ -22,8 +22,7 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef BITFN_H
-#define BITFN_H
+#pragma once
```

Comment:
> To hell with C standards, the long-term goal should be to eventually put that programming language in the grave.

That would make a very good tweet. But meanwhile the language is still here and living and just keep in mind that the list of compilers linked earlier are not the only tools that consume C code.

> add a section to the Makefile that detects the platform and uses sed to replace the pragmas with if-guards in the unfortunate cases when someone is trying to use that. Either that, or recommend cross-compiling.

Is that still in the "more serious note" ?